tests/cypress/support/commands.js

```diff
@@ -359,3 +359,103 @@ Cypress.Commands.add('logoutFromPublisher', () => {
+    cy.get('[data-testid="Publish-btn"]').click();
+
+    //check if the api is third-party and published
+    cy.get('[data-testid="itest-api-state"]').contains('PUBLISHED').should('exist');
+    cy.get('[data-testid="itest-third-party-api-label"]').contains('Third Party').should('exist');
+
+    //Check if the subscriptions,runtime config, resources, scopes, monetization, test console sections are present
+    cy.get('#itest-api-details-portal-config-acc').click();
+    cy.get('#left-menu-itemsubscriptions').should('exist');
+    cy.get('#left-menu-itemsubscriptions').click();
+    cy.get('[name="Unlimited"]').click();
+    cy.get('#subscriptions-save-btn').click();
+    cy.get('#itest-api-details-portal-config-acc').click();
+    cy.get('#itest-api-details-api-config-acc').click();
+    cy.get('#left-menu-itemRuntimeConfigurations').should('exist');
+    cy.get('#left-menu-itemresources').should('exist');
+    cy.get('#left-menu-itemLocalScopes').should('exist');
+    cy.get('#left-menu-itemMonetization').should('exist');
+    cy.get('#itest-api-details-api-config-acc').click();
+    cy.get('#left-menu-itemTestConsole').should('exist');
+
+    //Check if the api is not deployable
+    cy.get('#left-menu-itemdeployments').click();
+    cy.get('[data-testid="third-party-api-deployment-dialog"]').contains('This API is marked as a third party API. The requests are not proxied through the gateway. Hence, deployments are not required.').should('exist');
+    cy.get('#deploy-btn').should('be.disabled');
+
+    //Check if prompts when switching to a regular api
+    cy.get('#itest-api-details-portal-config-acc').click();
+    cy.get('#left-menu-itemDesignConfigurations').click();
+    cy.get('[name="advertised"]:last').click();
+    cy.get('[data-testid="itest-update-api-confirmation"]').should('exist');
+
+    cy.visit(`${Utils.getAppOrigin()}/publisher/apis`);
+    cy.get('[data-testid="itest-api-lifecycleState"] span').contains('PUBLISHED').should('exist');
+    cy.get('[data-testid="third-party-api-card-label"]').should('exist');
+
+})
+
+Cypress.Commands.add('viewThirdPartyApi', (apiName = null) => {
+    cy.get('[area-label="Go to ThirdPartyApi"]').click();
+
+    //Check if the subscriptions, tryout, comments and SDKs sections are present
+    cy.get('#left-menu-credentials').should('exist');
+    cy.get('#left-menu-test').should('exist');
+    cy.get('#left-menu-comments').should('exist');
+    cy.get('#left-menu-sdk').should('exist');
+
+    //Visit Original Developer Portal
+    cy.get('#left-menu-credentials').click();
+    cy.get('[data-testid="itest-original-devportal-link"]').should('exist');
+    cy.get('[data-testid="itest-no-tier-dialog"]').contains('No tiers are available for the API.').should('exist');
+
+    //Check if authorization header and value can be customized
+    cy.get('#left-menu-test').click();
+    cy.get('#advAuthHeader').should('exist');
+    cy.get('#advAuthHeaderValue').should('exist');
+
+})
+
+
```

Comment:
Remove additional new lines. Only 1 new line would be enough.

Reply:
This is fixed in [#380](https://github.com/wso2/apim-apps/pull/380)